modules/text_server_fb/text_server_fb.h

```diff
@@ -463,16 +464,33 @@ class TextServerFallback : public TextServerExtension {
 	mutable RID_PtrOwner<FontFallbackLinkedVariation> font_var_owner;
 	mutable RID_PtrOwner<FontFallback> font_owner;
 	mutable RID_PtrOwner<ShapedTextDataFallback> shaped_owner;
+	mutable Mutex rid_lock;
```

Comment:
```suggestion
	mutable Mutex rid_mutex;
```
Descriptive name IMO